sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -628,6 +630,12 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    *   spark.read().text("/path/to/spark/README.md")
    * }}}
    *
+   * You can set the following text-specific options to deal with text files:
+   * <ul>
+   * <li>`lineSep` (default is `\r\n` or `\n`): defines the line separator that should
```

Comment:
It sounds tricky but I guess we don't want a platform-dependent one - [SPARK-18076](https://issues.apache.org/jira/browse/SPARK-18076). Up to my knowledge, `LineRecordReader` should cover both `\r\n` and `\n`.  I will try to improve this documentation when cleaning other comments together if we are okay with this.